### PR TITLE
A11y fixes

### PR DESCRIPTION
**Pull request by iamjolly**
## Description

Adds a number of minor accessibility adjustments/fixes to the site as a first pass. 

Fixes the following issues:
* Top banner: Logo alt text (accessible name) in header overridden by link’s aria-label value of “Home”
* Top banner: Remove title=”home” from anchor element wrapping logo
* Top banner: Remove word “Logo” from the alt text for logo image
* Top banner: role=”banner” redundant on <header> element, recommend removing that role.
* Top banner: Main nav current item would benefit from having an aria-current=page indicator.
* Home page: Alt text for the middle image for Learning and development (woman writing on whiteboard) is copied from the first image (young man in business suit…)
* Footer: site navigation links and important links nav - consider using unordered lists for nav links in each.
* Footer: follow us should not be an H1. Consider making it consistent with Contact us.
* Who we are/About page: Reconsider use of word “attractive” in alt text for the Inclusion and Impact images of people. Consider less other descriptors like “smiling…” or “happy”
* Opportunity page: Consider removing “logo” from alt text of the GSA, PIF, USDS, and 18F logo images.
* _Partially_ addresses heading level issues (multiple h1s, adding some heading level hierarchy/nesting) on the About page.

## Checklist

- [x] My code follows the existing style, code structure, and naming taxonomy
- [x] I have performed a self-review of my own code and have no open issues
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or syntax errors

### CODE REVIEW

**Comment by DaveSkender** [comment]
Hey @iamjolly, can you target the `initial-site` branch instead of `main`?  It's the most up-to-date one and probably contains some of these fixes anyway.

File: pages/about/index.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h1>
       <div class="box about-whoweare-image"></div>
       <div class="box box--primary-vivid">
         <div class="box-content">
-          <h1>Who We Are</h1>
```
Comment (DaveSkender) [inline pages/about/index.html:46]: I think I get what you're going for here, that `h1` should be limited to a once-per-page topper type thing; however, this `box` has unique font weights, so switching to `as-h1` style is not the right style.  What I can do, is just refactor the `h2` in `box` to look like the `box h1` and just not use the `as-h1` style.

File: _includes/footer.html [hunk 2]
```diff
@@ -68,9 +68,9 @@
             </li>
 
             <!-- social-media -->
-            <h1 class="text-white font-sans-2xl text-medium margin-y-2">Follow us</h1>
```
Comment (DaveSkender) [inline _includes/footer.html:71]: I had addressed this partly in the `initial-site` branch, but think your use of `<p>` sounds more appropriate based on your other changes.

File: _includes/home.html [hunk 3]
```diff
@@ -87,7 +87,7 @@ <h2 class="twoline"> High-impact <br class="display-none tablet:display-inline"
       <div class="benefit">
         {% asset
         "pexels-christina-morillo-1181398.jpg"
-        alt="young man in business suit explaining in conference room" %}
```
Comment (DaveSkender) [inline _includes/home.html:90]: I fixed this in `initial-site`, but probably with not-as-good `alt` text.

**Review by DaveSkender — COMMENTED** [review]
Some of these I may have to refactor (e.g. convert the `h1` to `h2`), but all the others look good to me.

**Comment by DaveSkender** [comment]
I'm gonna do a little refactoring on your branch and rebaseline to `initial-site`, and try to address your full list.  Then, have you take a quick look at it to check my work.  Standby.

I also see some of your changes broke the built-in a11y automated tests for aria-labels; I'll take a look at those too, could be that test package is out of date on standards.

**Comment by iamjolly** [comment]
@DaveSkender - This all sounds good. Let me know if you want to connect today to talk through anything. I have time!

File: pages/about/index.html (hunk 1 above)
Comment (iamjolly) [reply pages/about/index.html:46]: Yep, this is more nuanced and worth thinking through a bit. Generally, I like to approach it as one `h1` per page, with the site title/logo being an `h1` on the home page only, then the page title or main topic being the `h1` on interior site pages. 

I didn't want to dig into or tamper with the CSS too much as you're more familiar with how you were setting up the sections and the classes to support them. 

If this helps, here are two pretty good resources for heading structure that I like:
* [Nomensa: How to structure headings for web accessibility](https://www.nomensa.com/blog/how-structure-headings-web-accessibility)
* [W3C: Page Structure > Headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)

**Review by DaveSkender — APPROVED** [review]
I think we're good to go on these; however the build system is having issues currently -- once those are resolved I'll merge to our open `initial-site` branch.

**Pull request merged**